sdk-rs/src/websocket_program_account_subscriber.rs

```diff
@@ -116,49 +116,92 @@ where
 
         let event_emitter = self.event_emitter.clone();
 
-        local.run_until(async move {
+        // local.run_until(async move {
+        //     let (mut accounts, unsubscribe) = pubsub.program_subscribe(
+        //         &drift_program::ID,
+        //         Some(config)
+        //     ).await.unwrap(); 
             
-            let (mut accounts, unsubscribe) = pubsub.program_subscribe(
+        //     self.unsubscriber = Some(unsubscribe);
+            
+        //     while let Some(message) = accounts.next().await {
+        //         let slot = message.context.slot;
+        //         if slot >= latest_slot {
+        //             latest_slot = slot;
+        //             let pubkey = message.value.pubkey;
+        //             let account_data = message.value.account.data;
+        //             let on_update_clone = on_update.clone();
+        //             match Self::decode(account_data) {
+        //                 Ok(data) => {
+        //                     let data_and_slot = DataAndSlot { slot, data };
+        //                     if let Some(ref on_update_callback) = on_update_clone {
+        //                         on_update_callback(event_emitter.clone(), pubkey, data_and_slot);                          
+        //                     }
+        //                 },
+        //                 Err(e) => {
+        //                     error!("Error decoding account data: {e}");
+        //                 }
+        //             }
+        //         } else {
+        //             warn!("Received stale data from slot: {slot}");
+        //         }
+        //     }
+
+        // }).await;
+
+        let (unsub_tx, mut unsub_rx) = tokio::sync::mpsc::channel::<Option<()>>(1);
```

Comment:
don't think the Option is needed?
```suggestion
        let (unsub_tx, mut unsub_rx) = tokio::sync::mpsc::channel::<()>(1);
```